src/test/java/com/example/music_player/storage/CloudStorageAmazonS3Test.java

```diff
@@ -24,29 +18,23 @@
 import java.nio.charset.StandardCharsets;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.ArgumentMatchers.anyString;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.mockito.Mockito.*;
 
-//@SpringBootTest(properties = "application-test.properties", classes = {MusicPlayerApplication.class} )
 @ExtendWith(MockitoExtension.class)
 public class CloudStorageAmazonS3Test {
```

Comment:
Add case, when exception occurs somewhen after tempFIle is created and before tempFile is deleted. And check that tempFile is actually deleted, otherwise it is resource leak problem.